### DISCUSSION TITLE
Release of fastapi-users 11.X

### DISCUSSION THREAD

**Discussion by caniko** [fastapi-users/fastapi-users#1199] [Q&A] [answered]
Hello @frankie567, could you please outline the release scheduling for 11.X? I am stuck on `10.1.3` while I am waiting for you to add #1037 back. You are adding this fix back in 11.X right?

Also, are you sure this counts as a breaking change considering it was a bug in the first place. [jupyverse](https://github.com/jupyter-server/jupyverse/blob/main/pyproject.toml) no longer uses `fastapi-users` anyway, so I don't see why this would hurt.

Thanks.

**Comment by frankie567** [comment, marked as answer]
I'll revive #1049, that should fix the problems raised in #1037. It solves lot of ambiguities by always building full response objects. It'll definitely be a breaking change in this context, that will need a major version bump.

Regarding jupyverse, they definitely [still use fastapi-users](https://github.com/jupyter-server/jupyverse/blob/main/plugins/auth/fps_auth/backends.py), but I think I have a solution that'll work for them.

**Reply by caniko** [reply]
I thought it wasn't in use because of `fastapi-user`s absence from the `pyproject.toml` file. Too quick on the trigger, apologies for the confusion
